v0.7/language/bert/mlperf.conf

```diff
@@ -0,0 +1,52 @@
+# The format of this config file is 'key = value'.
+# The key has the format 'model.scenario.key'. Value is mostly int64_t.
+# Model maybe '*' as wildcard. In that case the value applies to all models.
+# All times are in milli seconds
+
```

Comment:
Can we move this 1 level up so it is good for all models under v0.7 ?

Reply:
done